src/main/scala/akka/contrib/datareplication/ORMultiMap.scala

```diff
@@ -7,21 +7,21 @@ import akka.cluster.{ UniqueAddress, Cluster }
 
 object ORMultiMap {
 
+  val _empty: ORMultiMap[Any] = new ORMultiMap(ORMap.empty)
   /**
    * Provides an empty multimap.
    */
-  val empty: ORMultiMap =
-    new ORMultiMap(ORMap.empty)
-  def apply(): ORMultiMap = empty
+  def empty[A]: ORMultiMap[A] = _empty.asInstanceOf[ORMultiMap[A]]
+  def apply(): ORMultiMap[Any] = _empty
 
   /**
    * Java API
    */
-  def create(): ORMultiMap = empty
+  def create[A](): ORMultiMap[A] = empty[A]
 
   def unapply(value: Any): Option[Map[String, Set[Any]]] = value match {
```

Comment:
then it must be something like `def unapply[A](value: ReplicatedData): Option[Map[String, Set[A]]]`
but I can't get that to work with for example:

      def receive = {
          case Changed("key", ORMultiMap(entries)) =>
      }

then the `entries` is a `Map[String, Set[Nothing]]`
which is worse than current `unapply` that gives at least Map[String, Set[Any]]

and I don't know if it's possible to define the `A` in the pattern match. This does not work:

          case Changed("key", ORMultiMap[Int](entries)) =>